Fix issue #70 : Error when removing status folder

mantoine opened the pull request:
- replace path "." by **DIR** in status/index.php
- add defined('APPLICATION_ENV') in public/index.php
- modify algorithm of check acl in ../library/Centurion/Tool/Project/Provider/Install.php :
  -- add a feature let the script to fix acl to file and path with \* selector

mantoine committed:
Fix issue #70 : Error when removing status folder
- replace path "." by __DIR__ in status/index.php
- add defined('APPLICATION_ENV') in public/index.php
- modify algorithm of check acl in ../library/Centurion/Tool/Project/Provider/Install.php :
-- add a feature let the script to fix acl to file and path with * selector

lchenay commented:
Thanks mantoine for this fix.
I will merge it, but before:
- **DIR** can not be used: its a 5.3 constant that does not exists in 5.2. I will change it to dirname(**FILE**)
- why have you make a second foreach? is it because you want to manage the "*"?

mantoine commented:
Hi,
there is no problem for your change;
Indeed, i made a second foreach to manage file, and \* selector and not 
only folder.
maybe there is a best way to do that, i do this change this morning.

Le 03/05/2012 14:00, Laurent a écrit :

> Thanks mantoine for this fix.
> I will merge it, but before:
> - **DIR** can not be used: its a 5.3 constant that does not exists in 5.2. I will change it to dirname(**FILE**)
> - why have you make a second foreach? is it because you want to manage the "*"?
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/centurion-project/Centurion/pull/77#issuecomment-5485979

## 

Cordialement,

Michael Antoine
Ingénieur Développement et Architecte junior
Nouvelles Technologies

mantoine committed:
merge of two foreach :
- one was for manage folder path
- the other one was for manage file and '*' selector
